packages/components/src/internal/app/utils.ts

```diff
@@ -457,6 +457,11 @@ export function isCustomImportTemplatesEnabled(moduleContext?: ModuleContext): b
     return isFeatureEnabled(ProductFeature.CustomImportTemplates, moduleContext);
 }
 
+// Everything except sample manager gets conditional formatting.
+export function isConditionalFormattingEnabled(moduleContext?: ModuleContext): boolean {
+    return isFeatureEnabled(ProductFeature.ConditionalFormatting) || !isSampleManagerDistribution(moduleContext);
```

Comment:
I'm really confused by this isSampleManagerDistribution part of the check. With my localhost server if I have the premium module, there is no way to get this function to return false (i.e. any product configuration I select won't matter since the second part of the check will always be true). Does that mean that if a distribution has the premium module they should always see conditional formatting enabled?

Reply:
Correct, if you have the `premium` module conditional formatting is available to you. LKSM-only distributions do not include the `premium` module, and those are the ones we're trying to exclude this feature from.